src/openms/include/OpenMS/QC/Suitability.h

```diff
@@ -0,0 +1,173 @@
+    /// Constructor
+    /// Settings are initialized with their default values:
+    /// no_re_rank = false, novo_fract = 1, FDR = 0.01
+    Suitability();
+
+    /// Destructor
+    ~Suitability() = default;
+
+    /**
+    * @brief Computes suitability of a database used to search a mzML
+    *
+    * Counts top deNovo and top database hits. The ratio of db hits vs
+    * all hits yields the suitability.
+    * To re-rank cases, where a de novo peptide scores just higher than
+    * the database peptide, a decoy cut-off is calculated. This functionality
+    * can be turned off. This will result in an underestimated suitability,
+    * but it can solve problems like different search engines or to few decoy hits.
+    * 
+    * Parameters can be set using the functionality of DefaultParamHandler.
+    * Parameters are:
+    *                no_re_rank - re-ranking can be turned off with this
+    *                novo_fract - percent of deNovo peptides to capture in re-ranking
+    *                FDR        - q-value that should be filtered for
+    *
+    * Result is appended to the result member. This allows for multiple usage.
+    *
+    * Attention: This function will calculate q-values for all peptide hits. Your input
+    *            will not be the same afterwards.
+    *
+    * @param pep_ids      vector containing pepIDs coming from a deNovo+database 
+    *                     identification search without FDR (currently only Comet-support)
+    * @throws             MissingInformation if no target/decoy annotation is found
+    * @throws             MissingInformation if no xcorr is found
+    * @throws             Precondition if a q-value is found in the input
+    */
+    void compute(std::vector<PeptideIdentification>& pep_ids);
+
+    /// return results
+    const std::vector<SuitabilityData>& getResults() const;
+
+  private:
+    /// result vector
+    std::vector<SuitabilityData> results_;
+
+    /**
+    * @brief Calculates the xcorr difference between the top two hits marked as decoy
+    *
+    * Only searches the top ten hits for two decoys. If there aren't two decoys, DBL_MAX
+    * is returned.
+    *
+    * @param pep_id     pepID from where the decoy difference will be calculated
+    * @returns          xcorr difference
+    * @throws           MissingInformation if no target/decoy annotation is found
+    * @throws           MissingInformation if no xcorr is found
+    */
+    double getDecoyDiff_(const PeptideIdentification& pep_id);
+
+    /**
+    * @brief Calculates a xcorr cut-off based on decoy hits
+    *
+    * Decoy differences of all N pepIDs are calculated. The (1-novo_fract)*N highest
+    * one is returned.
+    * It is asssumed that this difference accounts for novo_fract of the re-ranking cases.
+    *
+    * @param pep_ids      vector containing the pepIDs
+    * @param novo_fract   fraction of how many cases, where a de novo peptide scores just higher than the database peptide, will be re-rank
```

Comment:
I still find this misleading, because this function only cares about decoy scores and has nothing to do with deNovo.
Rephrase to what the parameter actually does (and maybe rename it)?

Or maybe write both explanations, saying that the current one is the one from the paper?

Reply:
do this only in this function or change all occurrences?